packages/slice-machine/pages/changes.tsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import { Box, Button, Flex, Spinner, Text, useThemeUI } from "theme-ui";
+import Container from "components/Container";
+import Header from "components/Header";
+import { MdModeEdit } from "react-icons/md";
+import { useSelector } from "react-redux";
+import { getLibraries, getRemoteSlices } from "../src/modules/slices";
+import { SliceMachineStoreType } from "../src/redux/type";
+import SliceMachineIconButton from "../components/SliceMachineIconButton";
+import { TextWithGreyBackground } from "../components/Text/TextWithGreyBackground";
+
+const PushChangesButton = ({
```

Comment:
OPT:
I think each component deserve its own file.
this can definitely be done later when the logic will be done for the button.

Reply:
Agreed!